src/acquisition/covidcast/database.py

```diff
@@ -327,12 +335,12 @@ def update_covidcast_meta_cache(self, metadata):
 
     self._cursor.execute(sql, (epidata_json,))
 
-  def retrieve_covidcast_meta_cache(self):
+  def retrieve_covidcast_meta_cache(self, table_name='covidcast_meta_cache_test'): # TODO: remove arg
```

Comment:
Why is this defaulting to the test table rather then the normal one (compared to the method above which defaults to the existing one)

Reply:
this is for keeping things consistent when dealing with the temp table...  im essentially side stepping the regular cache except for edge cases, and things will go back to a less complicated state when we decide we like this type of caching. 
 the method above existed already and the default value preserves the previous behavior.

Reply:
heres what it looks like when its been cleaned up to remove the temp table: https://github.com/cmu-delphi/delphi-epidata/pull/377